recipes/tokyocabinet/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+./configure --prefix=$PREFIX \
+	    --with-zlib=$PREFIX \
+	    --with-bzip=$PREFIX
+
+make
+make check > /dev/null
```

Comment:
So you can see the whole log by clicking the `Raw log` button. It shows you in plain text, but it included more than 10,000 lines. In any event, I understand the need to not pollute the log. This trick may split the difference. Basically it only writes out the test results if testing failed, which is basically what one typically wants.

```bash
make check &> make_check.log || { cat make_check.log; exit 1;}
```